content/en/docs/tutorials/online-training/overview.md

```diff
@@ -63,6 +63,8 @@ Here are some of the sites that offer online training for Kubernetes:
 
 * [Microservice Applications in Kubernetes - 100% Hands-On Experience (Linux Academy)] (https://linuxacademy.com/devops/training/course/name/learn-microservices-by-doing)
 
+* [Magic Sandbox - an interactive, hands-on Kubernetes learning platform](https://msb.com)
```

Comment:
Most of the other lines have the course name followed by company name in brackets.

Also, this line has “an interactive, hands-on Kubernetes learning platform” which seems a bit vendor pitch-y to me.

Reply:
Plus, it's sorted wrong. Magic comes before Microservice.

Reply:
Thanks for the comments! Happy to change the description to something that fits better, if you have any suggestions. I'm gonna sign the CLA now